plugins/techdocs-backend/config.d.ts

```diff
@@ -198,7 +198,12 @@ export interface Config {
            */
           azureBlobStorage?: {
             /**
-             * (Required) Credentials used to access a storage container.
+             * (Optional) Connection string of the storage container.
+             * @visibility secret
+             */
+            connectionString: string;
+            /**
+             * (Optional) Credentials used to access a storage container.
```

Comment:
Oh this makes me realize. You now changed whether they are optional or not. The `config.d.ts` needs to mark both `connectionString?:` and `credentials?:` like that with a question mark, since otherwise the config might enforce that they BOTH are always specified.

This should be the very last fix...

Reply:
Done ! Please check.